### ISSUE TITLE
tmux session on localhost is misinterpreted as telnet session

### ISSUE THREAD

**Issue by XDex** [nojhan/liquidprompt#377]
Hi!
I'm having the same issue as #304 on Debian Testing.
Relevant output:
```
⏚ [andrei@andrei-lt-deb:~] $ echo $(who am i | sed -n 's/.*(\(.*\))/\1/p')
9574).%0
⏚ [andrei@andrei-lt-deb:~] $ tmux -V
tmux 2.0
⏚ [andrei@andrei-lt-deb:~] $ who am i
andrei   pts/1        2015-06-24 17:21 (tmux(9574).%0)
```

**Comment by dolmen** [comment]
Do you use the `develop` branch?

**Comment by XDex** [comment]
Nope, should I?

**Comment by dolmen** [comment]
Yes, you should as it includes many fixes, and some are related to tmux. See [the CHANGES on develop](https://github.com/nojhan/liquidprompt/blob/develop/CHANGES) since the last release.

I'm only considering bug reports against the `develop` branch.

**Comment by dolmen** [comment]
Note for example that #370 (which is somewhat related to tmux) has been fixed on the `develop` branch.

**Comment by XDex** [comment]
Just switched to develop. The issue persists..

**Comment by XDex** [comment]
Fixed in PR #383

**Comment by dolmen** [comment]
As I rejected the "fix" in #383, the issue is still open.

**Comment by dolmen** [comment]
Closing because duplicate of #304.
